README.md

```diff
@@ -17,7 +17,7 @@ This will happen if :
 ## Configurations
 Please use `ZipkinConig` class to configure the module and verify these values and modify them according to your service/environment.
 
-- `Enable` - true: enable ZipkinMiddleware/ZipkinMessageHandler, false: disable ZipkinMiddleware/ZipkinMessageHandler.
+- `BypassMode` - false: enable ZipkinMiddleware/ZipkinMessageHandler, enable: disable ZipkinMiddleware/ZipkinMessageHandler.
```

Comment:
Maybe you wanted to write

**true***: disable ZipkinMiddleware/ZipkinMessageHandler*

?